untitled.js

```diff
@@ -859,19 +899,19 @@ function handle_improveskill(args, msg) {
 	var d = new Date();
 	var n = d.toLocaleString();
 	var ml = parseInt(myGet(skill_att_name.slice(0,-4)+"ML",char.id,0));
-	var logout = myGet("TEXTAREA_LOG",char.id,"");
+
 	roll = randomInteger(100) + parseInt(myGet(skill_att_name.slice(0,-4)+"SB",char.id,0));
 	if (roll >= ml) {
 		mySet(skill_att_name.slice(0,-4)+"ML",char.id,(ml+1));
 		sendChat("Skill Improvement " + myGet("NAME",char.id,""), "<br>" 
 			+ "<br>" + " roll " +roll +": SUCCESS<br>" + args[2] + " ML increases to " + (ml+1));
-		mySet("TEXTAREA_LOG",char.id,logout + n + ":  " + getHarnTimeStr(state.MainGameNS.GameTime) 
-			+ ": Skill Improvement Roll: " + args[2] + " " + roll +": SUCCESS: ML = " + (ml+1) + "\n");
+		charLog(char.id, ": Skill Improvement Roll: " + args[2] + " " 
```

Comment:
the ` ` syntax for strings allows for substitution of variables with the ${} syntax. E.g. this could be written as
```charLog(char.id, `:Skill Improvement Roll: ${args[2]} ${roll}: SUCCESS: ML = ${ml+1)`, realtime, gametime);